REQUEST_EXAMPLES.md

```diff
@@ -5,44 +5,27 @@ This file contains a list of possible tweets that the bot can effectively read a
 **Using `#battbot` and mentioning `@battbot_` in every tweet request is mandatory.**
 
 ## Models
-The model names that you can use in a tweet (casing of the sentence won't matter) -
+The model names that you can use in a tweet (casing of the model won't matter) -
  - `Doyle-Fuller-Newman model` or `DFN`
  - `Single particle model` or `SPM`
  - `Single particle model with electrolyte` or `SPMe`
 
 ## Parameter sets / Chemistries
-The parameter sets or the chemistries that you can use in a tweet (casing of the sentence won't matter) -
+The parameter sets or the chemistries that you can use in a tweet (casing of the chemistry won't matter) -
  - `Chen2020`
  - `Marquis2019`
  - `Ai2020`
 
-## Compare 2 or more models with a constant discharge or with an experiment
-### Some examples -
-```
-@battbot_ compare SPM and SPMe for a constant dicharge of 1.25C at 290K with Chen2020 parameters #battbot.
-@battbot_ #battbot can you compare Single particle model, Single particle model with electrolyte and DFN model at 300K with a c-rate of 0.5C with Marquis2019 chemistry?
-@battbot_ #battbot compare spm, spme DFN model with a constant dicharge of 0.75C with Ai2020 at 280K
-@battbot #battbot compare spm and spme with chen2020 parameters with the experiment - [("Discharge at C/10 for 10 hours or until 3.3 V","Rest for 1 hour","Charge at 1 A until 4.1 V","Hold at 4.1 V until 50 mA","Rest for 1 hour")] * 3 at 298K
-```
-
-### Mandatory keywords -
-Adding ',', ':' or '-' anywhere and the casing of the sentence in the tweet text won't effect the simulation.
- - Adding the keyword `"Compare"`.
- For example -
- ```
- <space or no character>Compare<space or no character>
- Compare SPM and SPMe
- ```
- If not provided, the bot will give the following error -
- ```
- "I'm sorry, I couldn't understand the requested simulation."
- ```
+## Commom mandatory keywords -
```

Comment:
IMO I think this would be clearer if you just split into the two types of request (model or parameter comparison) and for each request provide the examples you have already and then tell the user what the required keywords are and what info they need to provide. E.g. for model comparison, you need the keyword "Compare" and then you need to provide some models, a parameter set, and then either a C-rate or experiment, and a temperature. 

The phrase "Common mandatory keywords" seems a bit odd, since if something is mandatory it must be common haha

Reply:
Yes, I think that would be better